packages/rendermime/style/base.css

```
@@ -32,18 +32,19 @@
 }
 
 .jp-RenderedText pre a:link {
-  text-decoration: none;
+  text-decoration: underline;
   color: var(--jp-content-link-color);
 }
 
 .jp-RenderedText pre a:hover {
   text-decoration: underline;
   color: var(--jp-content-link-color);
+  font-weight: bold;
 }
 
 .jp-RenderedText pre a:visited {
-  text-decoration: none;
-  color: var(--jp-content-link-color);
+  text-decoration: underline;
+  color: var(--jp-content-font-color3);
```

Can you define a semantic variable, e.g. `--jp-content-link-visited-color` or `--jp-content-visited-link-color` and use it where needed to allow overriding the colour choice and ensure consistency?

```suggestion
  color: var(--jp-content-link-visited-color);
```